include/SFML/Network/Ftp.h

```diff
@@ -168,7 +168,7 @@ CSFML_NETWORK_API size_t sfFtpListingResponse_getCount(const sfFtpListingRespons
 /// \brief Return a directory/file name contained in a FTP listing response
 ///
 /// \param ftpListingResponse Ftp listing response
-/// \param index              Index of the name to get (in range [0 .. getCount])
+/// \param index              Index of the name to get, in range [0 .. getCount() - 1]
```

Comment:
Seems like you're still missing @ChrisThrasher change request

```suggestion
/// \param index              Index of the name to get, in range [0 .. getCount())
```

Reply:
Ok, ok, that was intentional, but I'll accommodate to your liking.

`sf*Shape_{g,s}etPoint()` will still be using another notation, which is more explicit and clear, in my opinion.